faunadb-scala/src/main/scala/faunadb/values/MetricsResponse.scala

```diff
@@ -0,0 +1,10 @@
+package faunadb.values
+
+import faunadb.values.Metrics.Metrics
+
+case class MetricsResponse(metricsMap: Map[Metrics, Option[String]], value: Value) {
```

Comment:
Can we simply the type by removing the `Option` to have `metricsMap: Map[Metrics, String]`.
The value will be wrapped in an `Option` anyway when performing a `Map.get`.
WDYT?

Reply:
removed Option